lib/src/services/test_data/jira_api_data.ts

```diff
@@ -0,0 +1,144 @@
+import { readonlyDate } from "readonly-types";
+import { Board, CloudIssue } from "../jira";
+
+export const boardReturn: Board = {
+  id: 0,
+  name: "0",
+  type: "a really cool one",
+  columnConfig: {
+    columns: [
+      {
+        name: "backlog",
+        statuses: [
+          {
+            id: "0",
+          },
+        ],
+      },
+    ],
+    constraintType: "very constrained",
+  },
+};
+
+// eslint-disable-next-line functional/prefer-immutable-types
+export const searchJiraReturn: CloudIssue[] = [
```

Comment:
Extract any common fields from the objects in this array into another object, and then `...common` within each so that you don't have to duplicate lines.